R/FilterPermutation.R

```diff
@@ -0,0 +1,118 @@
+#' @title Permutation Filter
+#'
+#' @name mlr_filters_permutation
+#'
+#' @description
+#' Estimate how important individual features are by contrasting prediction
+#' performances. Compute the change in performance from permuting the values of
+#' a feature and compare that to the predictions made on the unmodified data.
```

Comment:
Not clearly documented on what data the learner is applied. Only on a single permuted col or on the complete task with one column permuted?

Reply:
The latter. Updated the description.